constants/materialStyles.ts

```diff
@@ -5,6 +5,7 @@ export const useStyles = makeStyles((theme: Theme) =>
       width: '90%',
       height: '100%',
       fontFamily: 'Roboto',
```

Comment:
was there a discussion about changing the font to Roboto? I think it confuses things a bit if we have too many fonts, but am happy to discuss as well.